tests/integration/bond_test.py

```diff
@@ -265,3 +265,81 @@ def test_set_bond_mac_address(eth1_up):
         eth1_cur_state = current_state[INTERFACES][1]
         assert bond99_cur_state[Interface.MAC] == MAC1.upper()
         assert bond99_cur_state[Interface.MAC] == eth1_cur_state[Interface.MAC]
+
+
+@pytest.fixture
+def bond99():
+    with bond_interface(name='bond99', slaves=[]) as bond_state:
+        yield bond_state
+
+
+def test_bond_and_test_inferfaces_have_different_macs(bond99):
```

Comment:
Had to get rid of it completely since I am attaching slaves eth1 and eth2 in the fixture itself making them have the same mac right there so assert macs not equal to each other fails since the bond and its slaves have same MAC right from the fixture